src/AlgebraicGeometry/Schemes/AffineSchemes/Morphisms/Attributes.jl

```diff
@@ -213,11 +239,22 @@ julia> (x1,x2,x3) = gens(R)
  x3
 
 julia> X = subscheme(Y, x1)
-Spec of Quotient of multivariate polynomial ring by ideal with 1 generator
-
-julia> f = inclusion_morphism(X, Y);
-
-julia> graph(f);
+Spectrum
+  of quotient
+    of multivariate polynomial ring in 3 variables over QQ
+    by ideal(x1)
+
+julia> f = inclusion_morphism(X, Y)
+Morphism
+  from [x1, x2, x3]  spec of quotient of multivariate polynomial ring
+  to   [x1, x2, x3]  affine 3-space over QQ
+given by the pullback function
+  x1 -> 0
+  x2 -> x2
+  x3 -> x3
+
+julia> graph(f)
+(Spec of quotient of multivariate polynomial ring, Morphism: spec of quotient of multivariate polynomial ring -> spec of quotient of multivariate polynomial ring, Morphism: spec of quotient of multivariate polynomial ring -> affine 3-space over QQ with coordinates x1, x2, x3)
```

Comment:
In one line mode any commas should be enclosed by ( ) This is to make sure that we can read a tuple. 
E.g. here: are x1, x2, x3) components of the tuple that is returned?

Reply:
Ah good point, I should removed the coordinates here